gen3/bin/kube-setup-cedar-wrapper.sh

```diff
@@ -0,0 +1,17 @@
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/lib/kube-setup-init"
+
+[[ -z "$GEN3_ROLL_ALL" ]] && gen3 kube-setup-secrets
+
+
+cedar_api_key_file="$(gen3_secrets_folder)/cedar_api_key.txt"
+
+if [[ ! -f cedar_api_key_file ]]; then
+    gen3_log_err "No CEDAR api key present in ${cedar_api_key_file}"
+else
+    g3kubectl create secret generic "cedar-service-api-key" --from-file=cedar_api_key.txt=${cedar_api_key_file}
```

Comment:
need to check existance of this secret and delete it if so, before creating a new one, otherwise this will error out

similiar to https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-portal.sh#L38-L41